ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml

```diff
@@ -110,6 +110,10 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-vsphere-serial
+  steps:
+    cluster_profile: vsphere
+    workflow: openshift-e2e-vsphere-serial
```

Comment:
do we need a workflow with this name somewhere?

Reply:
nvm, i found it!

Reply:
There is one, I think?  I don't know how this stuff is wired together.

Reply:
yeah there is, i found it here https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/vsphere/serial/openshift-e2e-vsphere-serial-workflow.yaml

just took me a few to figure out where it was lol